geode-protobuf/src/main/java/org/apache/geode/internal/protocol/Failure.java

```diff
@@ -26,7 +27,7 @@ public Failure(ClientProtocol.ErrorResponse errorResponse) {
     this.errorResponse = errorResponse;
   }
 
-  public static <T> Failure<T> of(ClientProtocol.ErrorResponse errorResponse) {
+  public static <T, V> Failure<T> of(ClientProtocol.ErrorResponse errorResponse) {
```

Comment:
Was this second generic parameter intended to be left here?

Reply:
A left over from a refactoring.... will amend